README.md

```diff
@@ -15,47 +15,35 @@ Add the plugin and repositories listing to `project.clj`. **NB: You need to add
 :plugins [[s3-wagon-private "1.2.0"]]
 ```
 
-You can store credentials either in an encrypted file or as
-environment variables. For the encrypted file, add this to
-`project.clj`:
+To authenticate to the S3 bucket, you can either:
 
-```clj
-:repositories [["private" {:url "s3p://mybucket/releases/" :creds :gpg}]]
-```
+- Use any of the AWS SDK [chained provider class][chained-provider-class] credential providers. An excerpt of the most commonly used credential providers:
+     - Environment Variables - `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`
+     - Java System Properties - aws.accessKeyId and aws.secretKey
```

Comment:
Updated the credentials profile section to link to the expected AWS file format and include how to to specify a certain profile. Below is the updated version of that section:

 - Credential profiles file at the default location (~/.aws/credentials) with the [AWS Credentials File Format](http://docs.aws.amazon.com/sdk-for-java/v1/developer-guide/credentials.html). To use a particular profile, specify the env var `AWS_PROFILE` or a Java system property of `aws.profile`, otherwise, the fallback will be the default profile name (`"default"`)